2018-10-15 12:51:56.379 -03:00 [Information] Applying migrations for Identity...
2018-10-15 12:51:59.403 -03:00 [Information] Migrations applied.
2018-10-15 12:51:59.404 -03:00 [Information] Filling database with system users and roles...
2018-10-15 12:51:59.832 -03:00 [Information] Database filled.
2018-10-15 12:52:02.020 -03:00 [Error] An internal error occurred [LogId:be03003c-4f5f-40b0-b08a-ed61f903b367].
System.InvalidOperationException: Conflicting schemaIds: Identical schemaIds detected for types Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Users.Post and Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Auth.Post. See config settings - "CustomSchemaIds" for a workaround
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaIdManager.IdFor(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.CreateReferenceSchema(Type type, Queue`1 referencedTypes)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.GetOrRegister(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateParameter(ApiDescription apiDescription, ApiParameterDescription apiParameterDescription, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.WhereSelectListIterator`2.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateOperation(ApiDescription apiDescription, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItem(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItems(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath, String[] schemes)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 12:52:51.350 -03:00 [Information] Applying migrations for Identity...
2018-10-15 12:52:52.281 -03:00 [Information] Migrations applied.
2018-10-15 12:52:52.281 -03:00 [Information] Filling database with system users and roles...
2018-10-15 12:52:52.697 -03:00 [Information] Database filled.
2018-10-15 12:52:54.660 -03:00 [Error] An internal error occurred [LogId:5272f0a6-c193-49bb-8d9c-d3b55c47c75a].
System.InvalidOperationException: Conflicting schemaIds: Identical schemaIds detected for types Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Users.Post and Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Auth.Post. See config settings - "CustomSchemaIds" for a workaround
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaIdManager.IdFor(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.CreateReferenceSchema(Type type, Queue`1 referencedTypes)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.GetOrRegister(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateParameter(ApiDescription apiDescription, ApiParameterDescription apiParameterDescription, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.WhereSelectListIterator`2.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateOperation(ApiDescription apiDescription, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItem(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItems(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath, String[] schemes)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 12:55:13.346 -03:00 [Information] Applying migrations for Identity...
2018-10-15 12:55:13.975 -03:00 [Information] Migrations applied.
2018-10-15 12:55:13.976 -03:00 [Information] Filling database with system users and roles...
2018-10-15 12:55:14.386 -03:00 [Information] Database filled.
2018-10-15 12:56:16.059 -03:00 [Information] Applying migrations for Identity...
2018-10-15 12:56:16.679 -03:00 [Information] Migrations applied.
2018-10-15 12:56:16.680 -03:00 [Information] Filling database with system users and roles...
2018-10-15 12:56:17.094 -03:00 [Information] Database filled.
2018-10-15 12:56:18.938 -03:00 [Error] An internal error occurred [LogId:23d64b8b-4675-444c-8f3f-21d1c012e397].
System.InvalidOperationException: Conflicting schemaIds: Identical schemaIds detected for types Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Users.Post and Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Auth.Post. See config settings - "CustomSchemaIds" for a workaround
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaIdManager.IdFor(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.CreateReferenceSchema(Type type, Queue`1 referencedTypes)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.GetOrRegister(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateParameter(ApiDescription apiDescription, ApiParameterDescription apiParameterDescription, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.WhereSelectListIterator`2.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateOperation(ApiDescription apiDescription, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItem(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItems(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath, String[] schemes)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 12:56:31.137 -03:00 [Information] Applying migrations for Identity...
2018-10-15 12:56:31.751 -03:00 [Information] Migrations applied.
2018-10-15 12:56:31.752 -03:00 [Information] Filling database with system users and roles...
2018-10-15 12:56:32.169 -03:00 [Information] Database filled.
2018-10-15 12:56:33.805 -03:00 [Error] An internal error occurred [LogId:fc7dd405-2a75-4e53-ba67-c4e43c83b219].
System.InvalidOperationException: Conflicting schemaIds: Identical schemaIds detected for types Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Users.Post and Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Dtos.v1.Auth.Post. See config settings - "CustomSchemaIds" for a workaround
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaIdManager.IdFor(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.CreateReferenceSchema(Type type, Queue`1 referencedTypes)
   at Swashbuckle.AspNetCore.SwaggerGen.SchemaRegistry.GetOrRegister(Type type)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateParameter(ApiDescription apiDescription, ApiParameterDescription apiParameterDescription, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.WhereSelectListIterator`2.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateOperation(ApiDescription apiDescription, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItem(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItems(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath, String[] schemes)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 13:00:47.066 -03:00 [Information] Applying migrations for Identity...
2018-10-15 13:00:48.040 -03:00 [Information] Migrations applied.
2018-10-15 13:00:48.040 -03:00 [Information] Filling database with system users and roles...
2018-10-15 13:00:48.454 -03:00 [Information] Database filled.
2018-10-15 13:07:21.414 -03:00 [Information] Applying migrations for Identity...
2018-10-15 13:07:23.205 -03:00 [Information] Migrations applied.
2018-10-15 13:07:23.206 -03:00 [Information] Filling database with system users and roles...
2018-10-15 13:07:23.635 -03:00 [Information] Database filled.
2018-10-15 13:12:15.789 -03:00 [Information] Applying migrations for Identity...
2018-10-15 13:12:17.094 -03:00 [Information] Migrations applied.
2018-10-15 13:12:17.095 -03:00 [Information] Filling database with system users and roles...
2018-10-15 13:12:17.510 -03:00 [Information] Database filled.
2018-10-15 13:17:09.016 -03:00 [Information] Username already exists.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.UserAlreadyExistsException: Username already exists.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost userPost) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 100
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 13:33:44.377 -03:00 [Information] Applying migrations for Identity...
2018-10-15 13:33:46.196 -03:00 [Information] Migrations applied.
2018-10-15 13:33:46.197 -03:00 [Information] Filling database with system users and roles...
2018-10-15 13:33:46.639 -03:00 [Information] Database filled.
2018-10-15 13:34:34.877 -03:00 [Error] An internal error occurred [LogId:9c6ee01d-62bf-4e47-afc4-dc85e477d498].
System.InvalidOperationException: Unable to resolve service for type 'Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Services.IEmailSender' while attempting to activate 'Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 13:46:09.182 -03:00 [Information] Applying migrations for Identity...
2018-10-15 13:46:11.027 -03:00 [Information] Migrations applied.
2018-10-15 13:46:11.028 -03:00 [Information] Filling database with system users and roles...
2018-10-15 13:46:11.433 -03:00 [Information] Database filled.
2018-10-15 13:46:45.739 -03:00 [Information] Username already exists.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.UserAlreadyExistsException: Username already exists.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost userPost) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 107
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 13:48:01.796 -03:00 [Information] Applying migrations for Identity...
2018-10-15 13:48:02.477 -03:00 [Information] Migrations applied.
2018-10-15 13:48:02.478 -03:00 [Information] Filling database with system users and roles...
2018-10-15 13:48:02.897 -03:00 [Information] Database filled.
2018-10-15 13:49:37.705 -03:00 [Information] One or more errors occurred during an Identity operation. Please check 'Items' for details.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.IdentityOperationException: One or more errors occurred during an Identity operation. Please check 'Items' for details.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost userPost) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 131
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 13:50:30.634 -03:00 [Information] One or more errors occurred during model validation. Please check 'Items' for details.
Furiza.AspNetCore.ExceptionHandling.ModelValidationException: One or more errors occurred during model validation. Please check 'Items' for details.
   at Furiza.AspNetCore.ExceptionHandling.ModelValidationAttribute.OnActionExecuting(ActionExecutingContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ModelValidationAttribute.cs:line 19
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 15:20:12.934 -03:00 [Information] Applying migrations for Identity...
2018-10-15 15:20:14.786 -03:00 [Information] Migrations applied.
2018-10-15 15:20:14.787 -03:00 [Information] Filling database with system users and roles...
2018-10-15 15:20:15.190 -03:00 [Information] Database filled.
2018-10-15 15:28:20.371 -03:00 [Information] Applying migrations for Identity...
2018-10-15 15:28:20.853 -03:00 [Information] Migrations applied.
2018-10-15 15:28:20.854 -03:00 [Information] Filling database with system users and roles...
2018-10-15 15:28:21.291 -03:00 [Information] Database filled.
2018-10-15 17:36:58.336 -03:00 [Information] Applying migrations for Identity...
2018-10-15 17:36:58.848 -03:00 [Information] Migrations applied.
2018-10-15 17:36:58.849 -03:00 [Information] Filling database with system users and roles...
2018-10-15 17:36:59.263 -03:00 [Information] Database filled.
2018-10-15 17:43:25.002 -03:00 [Information] One or more errors occurred during an Identity operation. Please check 'Items' for details.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.IdentityOperationException: One or more errors occurred during an Identity operation. Please check 'Items' for details.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 140
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:13:17.987 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:13:18.489 -03:00 [Information] Migrations applied.
2018-10-15 18:13:18.490 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:13:18.895 -03:00 [Information] Database filled.
2018-10-15 18:13:20.674 -03:00 [Error] An internal error occurred [LogId:dc47433d-59dc-4098-81f5-036737e7976a].
System.NotSupportedException: HTTP method "GET" & path "api/v1/Users" overloaded by actions - Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.Get (Furiza.AspNetCore.WebApiConfiguration.SecurityProvider),Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.ConfirmEmailAsync (Furiza.AspNetCore.WebApiConfiguration.SecurityProvider). Actions require unique method/path combination for Swagger 2.0. Use ConflictingActionsResolver as a workaround
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItem(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItems(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath, String[] schemes)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:13:59.549 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:14:00.040 -03:00 [Information] Migrations applied.
2018-10-15 18:14:00.041 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:14:00.438 -03:00 [Information] Database filled.
2018-10-15 18:14:02.100 -03:00 [Error] An internal error occurred [LogId:868d67c1-5fe9-4a27-91d3-822c05d24523].
System.NotSupportedException: HTTP method "GET" & path "api/v1/Users" overloaded by actions - Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.Get (Furiza.AspNetCore.WebApiConfiguration.SecurityProvider),Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.ConfirmEmailAsync (Furiza.AspNetCore.WebApiConfiguration.SecurityProvider). Actions require unique method/path combination for Swagger 2.0. Use ConflictingActionsResolver as a workaround
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItem(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItems(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath, String[] schemes)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:15:07.191 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:15:07.757 -03:00 [Information] Migrations applied.
2018-10-15 18:15:07.758 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:15:08.188 -03:00 [Information] Database filled.
2018-10-15 18:19:23.054 -03:00 [Information] One or more errors occurred during an Identity operation. Please check 'Items' for details.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.IdentityOperationException: One or more errors occurred during an Identity operation. Please check 'Items' for details.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.ConfirmEmailAsync(ConfirmEmailGet values) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 165
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:29:37.304 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:29:37.857 -03:00 [Information] Migrations applied.
2018-10-15 18:29:37.858 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:29:38.326 -03:00 [Information] Database filled.
2018-10-15 18:30:14.214 -03:00 [Information] Username already exists.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.UserAlreadyExistsException: Username already exists.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 107
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:33:47.609 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:33:48.100 -03:00 [Information] Migrations applied.
2018-10-15 18:33:48.101 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:33:48.515 -03:00 [Information] Database filled.
2018-10-15 18:38:19.999 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:38:20.523 -03:00 [Information] Migrations applied.
2018-10-15 18:38:20.524 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:38:20.975 -03:00 [Information] Database filled.
2018-10-15 18:38:42.743 -03:00 [Information] Username already exists.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.UserAlreadyExistsException: Username already exists.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 107
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:40:34.309 -03:00 [Information] One or more errors occurred during model validation. Please check 'Items' for details.
Furiza.AspNetCore.ExceptionHandling.ModelValidationException: One or more errors occurred during model validation. Please check 'Items' for details.
   at Furiza.AspNetCore.ExceptionHandling.ModelValidationAttribute.OnActionExecuting(ActionExecutingContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ModelValidationAttribute.cs:line 19
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:43:19.358 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:43:19.874 -03:00 [Information] Migrations applied.
2018-10-15 18:43:19.874 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:43:20.290 -03:00 [Information] Database filled.
2018-10-15 18:43:52.535 -03:00 [Error] An internal error occurred [LogId:4a3bb678-34d0-494b-a28c-5f7c4d265b34].
System.InvalidOperationException: The relative page path 'ConfirmEmail' can only be used while executing a Razor Page. Specify a root relative path with a leading '/' to generate a URL outside of a Razor Page.
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.CalculatePageName(ActionContext actionContext, String pageName)
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.Page(IUrlHelper urlHelper, String pageName, String pageHandler, Object values, String protocol, String host, String fragment)
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 123
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:44:36.130 -03:00 [Information] Username already exists.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.UserAlreadyExistsException: Username already exists.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 107
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:46:26.974 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:46:27.477 -03:00 [Information] Migrations applied.
2018-10-15 18:46:27.478 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:46:27.875 -03:00 [Information] Database filled.
2018-10-15 18:46:47.873 -03:00 [Information] Username already exists.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.UserAlreadyExistsException: Username already exists.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 107
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:46:59.707 -03:00 [Error] An internal error occurred [LogId:49b4ca47-8169-4192-86fb-44cbe84c56f7].
System.InvalidOperationException: The relative page path '~/Users/ConfirmEmail' can only be used while executing a Razor Page. Specify a root relative path with a leading '/' to generate a URL outside of a Razor Page.
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.CalculatePageName(ActionContext actionContext, String pageName)
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.Page(IUrlHelper urlHelper, String pageName, String pageHandler, Object values, String protocol, String host, String fragment)
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.Page(IUrlHelper urlHelper, String pageName, String pageHandler, Object values, String protocol)
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 123
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:54:01.767 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:54:02.292 -03:00 [Information] Migrations applied.
2018-10-15 18:54:02.293 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:54:02.709 -03:00 [Information] Database filled.
2018-10-15 18:55:21.715 -03:00 [Error] An internal error occurred [LogId:0f1a5ec6-503d-440d-85e6-b953e6bbf7ef].
System.InvalidOperationException: The instance of entity type 'ApplicationUser' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.ThrowIdentityConflict(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(TKey key, InternalEntityEntry entry, Boolean updateDuplicate)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(TKey key, InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.StartTracking(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState oldState, EntityState newState, Boolean acceptChanges)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState entityState, Boolean acceptChanges, Nullable`1 forceStateWhenUnknownKey)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.PaintAction(EntityEntryGraphNode node, Boolean force)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityEntryGraphIterator.TraverseGraph[TState](EntityEntryGraphNode node, TState state, Func`3 handleNode)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.AttachGraph(InternalEntityEntry rootEntry, EntityState entityState, Boolean forceStateWhenUnknownKey)
   at Microsoft.EntityFrameworkCore.DbContext.SetEntityState(InternalEntityEntry entry, EntityState entityState)
   at Microsoft.EntityFrameworkCore.DbContext.SetEntityState[TEntity](TEntity entity, EntityState entityState)
   at Microsoft.EntityFrameworkCore.DbContext.Attach[TEntity](TEntity entity)
   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.UpdateAsync(TUser user, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Identity.UserManager`1.UpdateUserAsync(TUser user)
   at Microsoft.AspNetCore.Identity.UserManager`1.ConfirmEmailAsync(TUser user, String token)
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.ConfirmEmailAsync(ConfirmEmailGet values) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 158
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 18:57:10.142 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:57:10.642 -03:00 [Information] Migrations applied.
2018-10-15 18:57:10.643 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:57:11.045 -03:00 [Information] Database filled.
2018-10-15 18:59:02.968 -03:00 [Information] Applying migrations for Identity...
2018-10-15 18:59:03.476 -03:00 [Information] Migrations applied.
2018-10-15 18:59:03.476 -03:00 [Information] Filling database with system users and roles...
2018-10-15 18:59:03.882 -03:00 [Information] Database filled.
2018-10-15 19:05:02.535 -03:00 [Information] Applying migrations for Identity...
2018-10-15 19:05:03.058 -03:00 [Information] Migrations applied.
2018-10-15 19:05:03.058 -03:00 [Information] Filling database with system users and roles...
2018-10-15 19:05:03.495 -03:00 [Information] Database filled.
2018-10-15 19:05:23.375 -03:00 [Error] An internal error occurred [LogId:8672efc5-cc46-46f4-9dfe-8fe97e12d67b].
System.InvalidOperationException: The relative page path 'ConfirmEmail' can only be used while executing a Razor Page. Specify a root relative path with a leading '/' to generate a URL outside of a Razor Page.
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.CalculatePageName(ActionContext actionContext, String pageName)
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.Page(IUrlHelper urlHelper, String pageName, String pageHandler, Object values, String protocol, String host, String fragment)
   at Microsoft.AspNetCore.Mvc.UrlHelperExtensions.Page(IUrlHelper urlHelper, String pageName, String pageHandler, Object values, String protocol)
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.UsersController.PostAsync(UsersPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\UsersController.cs:line 123
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:07:00.927 -03:00 [Information] Applying migrations for Identity...
2018-10-15 19:07:01.432 -03:00 [Information] Migrations applied.
2018-10-15 19:07:01.433 -03:00 [Information] Filling database with system users and roles...
2018-10-15 19:07:01.830 -03:00 [Information] Database filled.
2018-10-15 19:42:33.322 -03:00 [Information] Applying migrations for Identity...
2018-10-15 19:42:33.917 -03:00 [Information] Migrations applied.
2018-10-15 19:42:33.918 -03:00 [Information] Filling database with system users and roles...
2018-10-15 19:42:34.326 -03:00 [Information] Database filled.
2018-10-15 19:44:06.569 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:44:50.710 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:44:53.065 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:45:00.931 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:45:06.558 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:45:07.529 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:45:12.550 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:49:32.587 -03:00 [Information] Applying migrations for Identity...
2018-10-15 19:49:33.233 -03:00 [Information] Migrations applied.
2018-10-15 19:49:33.234 -03:00 [Information] Filling database with system users and roles...
2018-10-15 19:49:33.656 -03:00 [Information] Database filled.
2018-10-15 19:51:28.810 -03:00 [Information] Email confirmation is required in order to authenticate.
Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Exceptions.EmailConfirmationRequiredException: Email confirmation is required in order to authenticate.
   at Furiza.AspNetCore.WebApiConfiguration.SecurityProvider.Controllers.v1.AuthController.PostAsync(AuthPost model) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.WebApiConfiguration.SecurityProvider\Controllers\v1\AuthController.cs:line 69
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-15 19:59:57.450 -03:00 [Information] Applying migrations for Identity...
2018-10-15 19:59:59.408 -03:00 [Information] Migrations applied.
2018-10-15 19:59:59.409 -03:00 [Information] Filling database with system users and roles...
2018-10-15 19:59:59.810 -03:00 [Information] Database filled.
